pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -99,7 +99,10 @@
         "controller.minion.instances.cleanup.task.frequencyInSeconds";
     public static final String MINION_INSTANCES_CLEANUP_TASK_INITIAL_DELAY_SECONDS =
         "controller.minion.instances.cleanup.task.initialDelaySeconds";
-    public static final String TASK_METRICS_EMITTER_FREQUENCY_IN_SECONDS = "controller.minion.task.metrics.emitter.frequencyInSeconds";
+    public static final String MINION_INSTANCES_CLEANUP_TASK_MAX_OFFLINE_TIME_RANGE_SECONDS =
```

Comment:
I feel this should be named `minOfflineTimeSeconds`

Reply:
it should be max time as the threshold right? If the instance hasn't been back for more than this threshold, znodes should be dropped.

Reply:
So basically the minimum offline time before the instance being recollected rt?

Reply:
I am fine with `minOfflineTimeBerforeDeletion`